Doxyfile

```diff
@@ -789,13 +789,13 @@ WARN_IF_DOC_ERROR      = YES
 # EXTRACT_ALL is set to YES then this flag will automatically be disabled.
 # The default value is: NO.
 
-WARN_NO_PARAMDOC       = NO
+WARN_NO_PARAMDOC       = YES
```

Comment:
I feel like this is going to cause a lot of cases where a parameter or return value is self explanatory, but gets documented anyway, which would make the code a little bloated. What's your opinion?

Reply:
There was only 1.  And this PR "fixes" that one.